src/Components/NewBook.js

```diff
@@ -30,23 +30,34 @@ export default function NewBook() {
   };
 
   return (
-    <div>
-      <form className="form-container" onSubmit={handleSubmit}>
-        <input
-          type="text"
-          name="title"
-          value={bookData.title}
-          placeholder="Book title"
-          onChange={handleInputChange}
-        />
-        <input
-          type="text"
-          name="author"
-          value={bookData.author}
-          placeholder="Author"
-          onChange={handleInputChange}
-        />
-        <button type="submit">Add book</button>
+    <div className="container-lg ps-3">
+      <h2 className="fs-5 text-muted" style={{ fontFamily: 'Montserrat, sans-serif' }}>ADD BOOK</h2>
+      <form onSubmit={handleSubmit}>
+        <div className="row align-items-center">
+          <div className="col-md-7">
+            <input
+              className="w-100"
+              type="text"
+              name="title"
+              value={bookData.title}
+              placeholder="Book title"
+              onChange={handleInputChange}
+            />
+          </div>
+          <div className="col-md-3">
+            <input
+              className="w-100"
+              type="text"
+              name="author"
+              value={bookData.author}
+              placeholder="Author"
+              onChange={handleInputChange}
+            />
+          </div>
+          <div className="col-md-2">
+            <button className="btn btn-primary btn-sm w-100" type="submit">ADD BOOK</button>
+          </div>
+        </div>
```

Comment:
Kindly style your form section to match the given design as it is a requirement in this milestone that our project matches the given design😉.

yours
<img width="1436" alt="image" src="https://github.com/Innocent259/book-store/assets/64643995/dd74784c-d3de-4587-8265-acd626cc74d3">

design
<img width="813" alt="image" src="https://github.com/Innocent259/book-store/assets/64643995/79865562-feda-4ed0-923e-ecbf3a09b62c">

Kindly take note of the font properties, borders, width and height of each elements.